### PR TITLE
Text fixes in a few Learn CSS documents

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes: #7173 
Fixes: #7098 
Fixes: #6658 

Minor text snags, I also fixed some notecards, and the height on some examples to remove scrollbars while I was on these pages.

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html [hunk 1]
```diff
@@ -108,14 +109,13 @@ <h3 id="Controlling_inheritance">Controlling inheritance</h3>
...
+ <dd>Acts like {{cssxref("unset")}} in many cases, however will revert the property to the browser's default styling rather than the defaults applied to that property.</dd>
 </dl>
 
 <div class="notecard note">
-<p><strong>Note</strong>: There is also a newer value, {{cssxref("revert")}}, which has limited browser support.</p>
-</div>
-
-<div class="notecard note">
-<p><strong>Note</strong>: See {{SectionOnPage("/en-US/docs/Web/CSS/Cascade", "Origin of CSS declarations")}} for more information on each of these and how they work.</p>
+  <h4>Note:</h4>
```
Comment (teoli2003) [inline files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html:117]: In #7170, @wbamberg made all notes consistent through CSS (like in JS last month). This allows the md converter to parse them correctly, and also prevent yari to raise a _sectioning flaw_.

So `<p><strong>Note</strong></p>: See {{…` is the way to go for notes (and warnings, with the correct text).

File: files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html [hunk 2]
```diff
@@ -254,8 +255,9 @@ <h3 id="Specificity_2">Specificity</h3>
  <li>Selectors 5–7 are competing over the styling of the link's border when hovered. Selector six clearly loses to five with a specificity of 23 vs. 24 — it has one fewer element selectors in the chain. Selector seven, however, beats both five and six — it has the same number of sub-selectors in the chain as five, but an element has been swapped out for a class selector. So the winning specificity is 33 vs. 23 and 24.</li>
 </ul>
 
-<div class="note">
-<p><strong>Note</strong>: This has only been an approximate example for ease of understanding. In actuality, each selector type has its own level of specificity that cannot be overwritten by selectors with a lower specificity level. For example, a <em>million</em> <strong>class</strong> selectors combined would not be able to overwrite the rules of <em>one</em> <strong>id</strong> selector.</p>
+<div class="notecard note">
```
Comment (teoli2003) [inline files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html:258]: See comment above.

File: files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html [hunk 3]
```diff
@@ -277,8 +279,9 @@ <h3 id="!important">!important</h3>
  <li>The 2nd element <em>does</em> get the red background color, but no border. Why? Because of the <code>!important</code> declaration in the second rule — including this after <code>border: none</code> means that this declaration will win over the border value in the previous rule, even though the ID has higher specificity.</li>
 </ol>
 
-<div class="note">
-<p><strong>Note</strong>: The only way to override this <code>!important</code> declaration would be to include another <code>!important</code> declaration on a declaration with the <em>same specificity</em> later in the source order, or one with higher specificity.</p>
+<div class="notecard note">
```
Comment (teoli2003) [inline files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html:282]: See comment before

File: files/en-us/learn/css/building_blocks/debugging_css/index.html [hunk 4]
```diff
@@ -80,7 +80,8 @@ <h2 id="Adding_a_new_property">Adding a new property</h2>
 <p><img alt="The DevTools Panel, adding a new property to the rules, with the autocomplete for font- open" src="inspecting3-font-size.png" style="border-style: solid; border-width: 1px;"></p>
 
 <div class="notecard note">
-<p><strong>Note</strong>: There are other useful features in the Rules view too, for example declarations with invalid values are crossed out. You can find out more at <a href="/en-US/docs/Tools/Page_Inspector/How_to/Examine_and_edit_CSS">Examine and edit CSS</a>.</p>
+<h4>Note:</h4>
```
Comment (teoli2003) [inline files/en-us/learn/css/building_blocks/debugging_css/index.html:83]: See comment before.

File: files/en-us/learn/css/building_blocks/debugging_css/index.html [hunk 5]
```diff
@@ -98,7 +99,8 @@ <h2 id="Understanding_the_box_model">Understanding the box model</h2>
 <p><img alt="The Layout section of the DevTools" src="inspecting4-box-model.png" style="border-style: solid; border-width: 1px;"></p>
 
 <div class="notecard note">
-<p><strong>Note</strong>: Find out more in <a href="/en-US/docs/Tools/Page_Inspector/How_to/Examine_and_edit_the_box_model">Examining and Inspecting the Box Model</a>.</p>
+  <h4>Note:</h4>
```
Comment (teoli2003) [inline files/en-us/learn/css/building_blocks/debugging_css/index.html:102]: See comment before.

File: files/en-us/learn/css/building_blocks/sizing_items_in_css/index.html [hunk 6]
```diff
@@ -92,7 +92,8 @@ <h2 id="min-_and_max-_sizes">min- and max- sizes</h2>
 <p>This technique is used to make images <em>responsive</em>, so that when viewed on a smaller device they scale down appropriately. You should, however, not use this technique to load really large images and then scale them down in the browser. Images should be appropriately sized to be no larger than they need to be for the largest size they are displayed in the design. Downloading overly large images will cause your site to become slow, and it can cost users more money if they are on a metered connection.</p>
 
 <div class="notecard note">
-<p><strong>Note</strong>: Find out more about <a href="/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images">responsive image techniques</a>.</p>
+  <h4>Note:</h4>
```
Comment (teoli2003) [inline files/en-us/learn/css/building_blocks/sizing_items_in_css/index.html:95]: See comment before.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Looks good except the technical changes about notes (made a few days ago by @wbamberg in preparation of the migration to Markdown). (see first comment for reference)

File: files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html (hunk 1 above)
Comment (rachelandrew) [reply files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html:117]: Then I would strongly suggest the style guide is changed https://developer.mozilla.org/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard as I've been changing then to the "new" way of doing them that was added to the style guide.

Comment (hamishwillee) [reply files/en-us/learn/css/building_blocks/cascade_and_inheritance/index.html:117]: Makes sense to me https://github.com/mdn/content/pull/7437

**Comment by rachelandrew** [comment]
@teoli2003 I fixed up the notes :)

**Comment by rachelandrew** [comment]
@teoli2003 could you re-look at this one for me?

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks!

**Pull request merged**